Isabel-Gomez/Precurso/Bingo/bingo.js

```diff
@@ -0,0 +1,147 @@
+let bingoCard = [
```

Comment:
Podrías hacer una función que cree un bingoCard con todos esos objetos hijos.

```suggestion

const createBingoCard = (childQuantity) => {
    let bingoCard = []
    for (let i = 0; i < childQuantity; i++) {
          bingoCard.push({ number: null, matched: false })
    }
    return bingoCard
}
```